ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
```

Comment:
Looks like we lost the ability to set kuryr networkType.

Reply:
It's there, the environment vairable KURYR_ENABLED is avialable to the hist step. It is defined in in the ref file  (ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml) here https://github.com/openshift/release/pull/11000/files#diff-a5cf4017a452b7856e949eeaa4afe2f9R16 with a default of false. 
Once this commit merges then we can go ahead and add support for KURYR by just having this script check for the value of KURYR_ENABLED.